syntax = "proto3";
option java_package = "com.huasheng.sa.quant.open.sdk.protobuf.hq.dto";
option java_outer_classname = "SecurityProto";

message Security {
  int32 dataType = 1; //股票类型
  string stockCode = 2;  //股票代码（行情接口：沙特市场股票代码需要带后缀 .SA）
}